src/backbone.syphon.valueassigners.js

```diff
@@ -0,0 +1,43 @@
+// Value Assigners
+// -------------------------
+
+// Value Assigners are used to  whether or not a
+// key should be assigned to a value, after the key and value have been
+// extracted from the element. This is the last opportunity to prevent
+// bad obj[key] from getting serialized to your object.
+
+var ValueAssignerSet = Syphon.ValueAssignerSet = TypeRegistry.extend();
+
+// Build-in Key Assignment Values
+var ValueAssigners = Syphon.ValueAssigners = new ValueAssignerSet();
+
+// return value by default
+ValueAssigners.registerDefault(function(value) {
+  return function(obj, key) {
```

Comment:
This feels like a complex implementation. Do you see developers adding their own custom `ValueAssigners` often?